src/index.tsx

```diff
@@ -16,7 +16,7 @@ export interface Props {
 
 const VisibilitySensor: React.FC<Props> = (props) => {
   const myView: any = useRef(null);
-  const [lastValue, setLastValue] = useState<boolean>(false);
+  const [lastValue, setLastValue] = useState<boolean | null>(null);
```

Comment:
Sorry, me again with some feedback. I'm trying to understand why we go for a double type declaration here. What's the scenario where setting this to `false` (instead of `null`), and removing line 29, that it would still show undesired behavior? 

Happy to merge this PR, just trying to wrap my head around why this would work.